_includes/banner.html

```diff
@@ -1,5 +1,5 @@
 <section class="os-banner">
-    <header class="banner-head" style="background-image: url('{{site.url}}{{site.baseurl}}/images/graphic-bg.jpg')">
+    <header class="banner-head" style="background-image: url('{{site.url}}{{site.baseurl}}/images/graphic-bg.png'); background-size: cover;">
```

Comment:
any reason why we should not add the background-size to CSS?

Reply:
No reason. Good catch, thank you. I'll update in another PR.

Reply:
Implemented in #2962